deps/build.jl

```diff
@@ -17,6 +17,8 @@ end
 
 provides(AptGet,"tcl8.5",tcl)
 provides(AptGet,"tk8.5",tk)
+provides(BSDPkg,"tcl85",tcl,os=:FreeBSD)
+provides(BSDPkg,"tk85",tk,os=:FreeBSD)
```

Comment:
this requires a minimum bindeps version of 0.6.0 https://github.com/JuliaLang/BinDeps.jl/commit/264e086062f3c39adba7997f187bd54bc0e6b218

you'll also need to drop julia 0.5 support as it's not longer supported